indy_node/server/config_req_handler.py

```diff
@@ -167,6 +175,16 @@ def validate(self, req: Request):
                                                               old_value="*",
                                                               new_value="*")])
 
+        elif typ == TXN_ATHR_AGRMT:
+            version = operation[TXN_ATHR_AGRMT_VERSION]
+            if self.state.get(":taa:v:{}".format(version), isCommitted=False) is not None:
```

Comment:
I would use helper methods (`indy_common/state/config.py`, see also `indy_common/state/domain.py`) to access state keys.